landside/catkin_ws/src/simulation/scripts/fake_cv_maker.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+from custom_msgs.msg import CVObject, SimObjectArray
+from tf import TransformListener
+from geometry_msgs.msg import Pose, Quaternion, PoseStamped
+import rospy
+from numpy import clip
+import resource_retriever as rr
+import yaml
+
+VERBOSE_BOUNDING_BOX = False
+
+def get_item_class(item_label):
+    """
+    Returns the type of a sim object given its label in the simulation.
+    Example: get_item_class("GateLeftChild#0") = GateLeftChild
+    """
+    hash_index = item_label.find('#')
+    if hash_index < 0: 
+        return item_label
+    return item_label[:hash_index]
+
+
+class BoundingBox:
+
+    def __init__(self):
+        rospy.init_node('sim_fake_cv_maker')
+        self.listener = TransformListener()
+        config_filepath = rr.get_filename(
+            'package://simulation/data/config.yaml', 
+            use_protocol=False
+        )
+        with open(config_filepath) as f:
+            data = yaml.safe_load(f)
+        self.publishers = {i.strip(): rospy.Publisher(f'/{i.strip().lower()}/left', CVObject, queue_size=10)
+                           for i in data['cv_objects']}
```

Comment:
This line also shows up in the add-sim-models branch, so we may have to change it there as well.

The point of `strip` here isn't clear to me besides just being extra safe, and removing it doesn't cause any problems.

There's no interface I know of that demands CV objects be published as lower case, besides just the fact that all of our current ROS topics are all lowercase (does ROS enforce this?). I wouldn't mind coming to an agreement with CV but at the moment I do not know who to talk to. I think that the names are put in PascalCase there because that's what the objects are called in CoppeliaSim. Really the config file would be clearest if each object type had a separate CoppeliaSim name and ROS topic name associated with it.

I extracted the initialization to a separate function and used a for loop.

Reply:
I plan to publish to the same topics as CV does, so for the gate, I'll publish to /cv/camera_left/start_gate (unless you want me to publish to some parallel channel like /fake_cv/camera_left/start_gate), and I'll make a mapping from the expected CoppeliaSim names to the expected topic names.

Reply:
Sounds good, so long as you and CV are clear on this.